### PR TITLE
Feat/#103 implement GET comment endpoint

### LINKED ISSUE #103: [TASK] Implement GET /comment/:listing_id
### Description

The response should be a list of Comments in chronological order (based on the **creation date**).

Each object in the list should contains:
- The comment ID
- The author ID
- author's name
- text
- time stamp
- isAuthor
- modified date (last udpate)

### Acceptance Criteria:
- Returns a list if comments exist
- Return `404 Not Found` if the `listing_id` does not exist in the DB
- Return an empty list if there is no comment

### Link

**User Story**: #102

### PR DESCRIPTION
**Pull request by wynonagcc**
# Description

Endpoint
- Implemented GET /comment/listing_id
- Doesn't require authentication
- Return 404 Not Found if the listing_id does not exist in the DB
- Return an empty list if there is no comment

Misc:
- Added `modified_date` column in the fujiji_comments table using the new migration script

Closes #103

### CODE REVIEW

File: fujiji-server/src/controllers/comment.js [hunk 1]
```diff
@@ -37,4 +41,25 @@ async function postComment(req, res, next) {
...
+    );
+  }
+
+  try {
+    const comments = await getComments(listingID);
+
+    return res.status(200).json({ comments });
+  } catch (err) {
+    return next(new APIError(err, 500));
+  }
```
Comment (devin-efendy) [inline fujiji-server/src/controllers/comment.js:62]: Can we also include these data from #103? I think we are missing the author's name and modified date.

Each object in the list should contains:
- The comment ID
- The author ID
- author's name
- text
- time stamp
- isAuthor
- modified date (last udpate)

I'm not sure if modified date column exist yet, though.

Comment (devin-efendy) [reply fujiji-server/src/controllers/comment.js:62]: Yes, it's the commenter's name. Can you also include the script for adding the column inside the migration-scripts directory? Thanks!

Comment (wynonagcc) [reply fujiji-server/src/controllers/comment.js:62]: For sure!

Comment (wynonagcc) [reply fujiji-server/src/controllers/comment.js:62]: Fixed!

**Review by devin-efendy — APPROVED** [review]

**Pull request merged**